repo/open.go

```diff
@@ -387,25 +510,31 @@ func formatBytesCachingEnabled(cacheDirectory string, validDuration time.Duratio
 	return validDuration > 0
 }
 
-func readRepositoryBlobBytesFromCache(ctx context.Context, cachedFile string, validDuration time.Duration) ([]byte, error) {
+func readRepositoryBlobBytesFromCache(ctx context.Context, cachedFile string, validDuration time.Duration) ([]byte, time.Time, error) {
```

Comment:
nit: can we add names to the output parameters, that time result is not obvious.

Reply:
done in [4dabb7d](https://github.com/kopia/kopia/pull/1758/commits/4dabb7d600c13aa983325619ac79ce3fcd271cdc)